packages/beacon-node/src/network/reqresp/reqResp.ts

```diff
@@ -45,12 +45,12 @@ export class ReqResp implements IReqResp {
   private libp2p: LibP2p;
   private readonly peersData: PeersData;
   private logger: ILogger;
-  private controller: AbortController;
   private reqRespHandlers: ReqRespHandlers;
   private metadataController: MetadataController;
   private peerRpcScores: IPeerRpcScoreStore;
   private inboundRateLimiter: IRateLimiter;
   private networkEventBus: INetworkEventBus;
+  private controller = new AbortController();
```

Comment:
Why do you need to initialize the AbortController on constructor?

Reply:
Not sure why this is showing as a diff here, but this is not newly introduced. See it on `unstable` branch. See [here](https://github.com/ChainSafe/lodestar/blob/f48bef0d7a2e6009cb8d4fbef9f5c2d1fc2ff264/packages/beacon-node/src/network/reqresp/reqResp.ts#L52)